src/test/scala/de/uhh/lt/jst/corpus/CoarsifyPosTagsTest.scala

```diff
@@ -0,0 +1,60 @@
+package de.uhh.lt.jst.corpus
+
+import java.nio.file.{Files, Paths}
+
+import com.holdenkarau.spark.testing.SharedSparkContext
+import de.uhh.lt.testtags.BrokenTest
+import org.scalatest._
+
+import scala.io.Source
+
+class CoarsifyPosTagsTest extends FlatSpec with Matchers with SharedSparkContext {
```

Comment:
Thank you for all the explanations. Yes, this is just exactly what I was curious about. 

This PR can be merged when the CoNLL issue will be resolved (#2 #1 at https://github.com/fmarten/JoSimText/issues/).